bootstrap.php

```diff
@@ -55,3 +55,46 @@
         'user_id'   => Cookie::get('logged_user_id'),
     ));
 });
+
+\Event::register_function('admin.beforeLogin', function ($params) {
+    $controller = \Arr::get($params, 'controllerInstance');
+    $wait_failures_config = \Config::load('novius_loginhistory::wait_after_admin_login_failures', true);
+    $wait_failures_enabled = \Arr::get($wait_failures_config, 'enabled', true);
+
+    if ($wait_failures_enabled) {
+        $time_from = \Date::forge()
```

Comment:
You're missing the concept of rate here, 3 failed attempts within 300 seconds will ban me for those same 300 seconds. The wait should start on the last failed attempt